src/common/Loader/Loader.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
```

Comment:
File name is weird. Also, this should be in components folder where all components from shadcn reside.